// Exceptions/Exceptions1.kt

fun erroneousCode() {
  // Uncomment this line to get an exception:
  // println("1$".toInt())
}

fun main(args: Array<String>) {
  erroneousCode()
}
